test/validators.js

```diff
@@ -1853,6 +1853,522 @@ describe('Validators', () => {
     });
   });
 
+  it('should validate passport number', () => {
```

Comment:
@profnandaa Check here for the tests,  I've already included test-cases, one per country code. It's a Large diff, it had be manually loaded, probably that's why you didn't notice it

Reply:
My bad missed that.